.pre-commit-config.yaml

```diff
@@ -71,21 +71,21 @@ repos:
     hooks:
       - id: isort
         name: Format imports
-        exclude: docs/source-app
+        exclude: docs/source
 
   - repo: https://github.com/psf/black
     rev: 22.12.0
     hooks:
       - id: black
         name: Format code
-        exclude: docs/source-app
+        exclude: docs/source
```

Comment:
Is this intended? Do we not want to just include the app files?

Reply:
unfortunalyly it is inted and it was explicitly asked by @williamFalcon as he was not happy about formating docs...
cc: @justusschock